assets/blocks/editor-components/limited-text-control/index.test.js

```diff
@@ -0,0 +1,180 @@
+/**
+ * External dependencies
+ */
+import { render, fireEvent } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import LimitedTextControl from './index';
+import userEvent from '@testing-library/user-event';
+
+describe( '<LimitedTextControl"/>', () => {
```

Comment:
Any specific reason for adding this `"` here?

Sounds a little weird to have the name of the set of the tests as `<LimitedTextControl"/>`.

```suggestion
describe( '<LimitedTextControl />', () => {
```